Shoko.Server/API/v3/Models/Shoko/WebUI.cs

```diff
@@ -383,8 +391,43 @@ public WebUISeriesFileSummary(SVR_AnimeSeries series, HashSet<EpisodeType> episo
                 .ThenBy(episode => episode.Number)
                 .Select(episode => new Episode.AniDB(episode.AniDB))
                 .ToList();
+
+
+            var releaseGroupNames = releaseGroups.Values
+                .Select(group => group.GroupNameShort)
+                .ToList();
+
+            var sourcesByType = filesWithXrefAndLocation
+                .Select(f => new
+                {
+                    f.file,
```

Comment:
why 8snf.file included here if it's not used? am i missing something?

Reply:
```suggestion
```

Reply:
"Oops". Spent a while going in circles to get this all to work and this is a remnant of my botched housekeeping.